specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -5351,6 +5383,10 @@
           "description": "The object representing the policy for taking backups on an account.",
           "type": "object",
           "$ref": "#/definitions/BackupPolicy"
+        },
+        "cors": {
```

Comment:
adding optional property also need version bump https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-changes-that-require-a-version-change

Reply:
adding property in output is breaking change, which requires version bump

Reply:
Hi @yungezz, this property has been available to users in production for some time, but was undocumented in swagger. We cannot add a new API version and restrict it as only available in that version because we could break users currently consuming the property. Should we add a new API version and document it there even if it remains available in previous versions?

Reply:
thanks for sharing, @shurd , just confirm, this PR is a fixing of Swagger bug actually right, not to introducing new property.

Reply:
Correct @yungezz